Sources/NIOCore/ByteBuffer-views.swift

```diff
@@ -34,37 +34,41 @@ public struct ByteBufferView: RandomAccessCollection, Sendable {
     /// Creates a `ByteBufferView` from the readable bytes of the given `buffer`.
     @inlinable
     public init(_ buffer: ByteBuffer) {
-        self = ByteBufferView(buffer: buffer, range: buffer.readerIndex ..< buffer.writerIndex)
+        self = ByteBufferView(buffer: buffer, range: buffer.readerIndex..<buffer.writerIndex)
     }
 
     @inlinable
     public func withUnsafeBytes<R>(_ body: (UnsafeRawBufferPointer) throws -> R) rethrows -> R {
-        return try self._buffer.withVeryUnsafeBytes { ptr in
-            try body(UnsafeRawBufferPointer(start: ptr.baseAddress!.advanced(by: self._range.lowerBound),
-                                            count: self._range.count))
+        try self._buffer.withVeryUnsafeBytes { ptr in
+            try body(
+                UnsafeRawBufferPointer(
+                    start: ptr.baseAddress!.advanced(by: self._range.lowerBound),
+                    count: self._range.count
+                )
+            )
         }
     }
 
     @inlinable
     public var startIndex: Index {
-        return self._range.lowerBound
+        self._range.lowerBound
     }
 
     @inlinable
     public var endIndex: Index {
-        return self._range.upperBound
+        self._range.upperBound
     }
 
     @inlinable
     public func index(after i: Index) -> Index {
-        return i + 1
```

Comment:
why are we losing the `return`s? That makes debugging so much harder because now you can't just add a `print` above :(